modules/network-observability-configuring-custom-metrics.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// network_observability/metrics-alerts-dashboards.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-configuring-custom-metrics_{context}"]
+= Configuring custom metrics
+. In the web console, navigate to the Import icon, +. 
+. Here are a couple of examples of `FlowMetrics` configurations you can add:
++
+.Generate a metric that tracks ingress bytes received from cluster external sources
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: flows.netobserv.io/v1alpha1
+kind: FlowMetric
+metadata:
+  name: flowmetric-cluster-external-ingress-traffic
+spec:
+  metricName: cluster_external_ingress_bytes_total  <1>
+  type: Counter                                     <2>
+  valueField: Bytes
+  direction: Ingress                                <3>
+  labels: [DstK8S_HostName,DstK8S_Namespace,DstK8S_OwnerName,DstK8S_OwnerType] <4>
+  filters:                                          <5>
+  - field: SrcSubnetLabel
+    matchType: Absence
+----
+<1> How the metrics appear, along with the prefix `netobserv`. 
+<2> The `type` `Counter` is useful for counting bytes or packets. Histogram is the other available type, which is useful for latencies. 
+<3> Direction of traffic to capture. Ingress is an alternative option. If not specified, both ingress and egress are captured. 
+<4> Labels define what the metrics look like and the relationship between the different entities. The labels also define the metrics cardinality. For example, `SrcK8S_Name` is a high cardinality metric.
```

Comment:
I think we need a warning or disclaimer here about high cardinality, telling that is can affect the memory usage of Prometheus.

On a related note, we have added this task https://issues.redhat.com/browse/NETOBSERV-1610 to add to the [flow format doc](https://docs.openshift.com/container-platform/4.15/observability/network_observability/json-flows-format-reference.html) whether each field is "suitable" for use in Prometheus (ie. is it high cardinality or not) => so perhaps would be good to link to the flow format reference here, to get more information about each field